.environment/gitleaks/gitleaks-config.toml

```diff
@@ -0,0 +1,97 @@
+title = "PRIME ReportStream Gitleaks Configuration"
+
```

Comment:
Some additional rules to consider:

- Azure
- Twilio
- SendGrid
- Okta

Reply:
Yep, I need to find out what their tokens look like so I can define a regex for them.

Reply:
Also needs IP(v4/6) address detection too

Reply:
SendGrid regex included added